src/interfaces/stepper-interfaces/YourCompany/business-contact-interface.ts

```diff
@@ -1,6 +1,6 @@
 export interface BusinessContactIF{
     email: string
-    confirmEmail: string
+    confirmEmail?: string
```

Comment:
Why is this optional? Isn't this the same scenario as for a correction?

Reply:
This isn't an actual value saved in the back end right? I think this prop is purely for front end validations.

Reply:
You're right. And I now think having it optional makes more sense.